### ISSUE TITLE
ST Stable 

### ISSUE THREAD

**Issue by NoAcctWanted** [yuliskov/SmartTube#3623]
I kept receiving the 403 error code on ST Beta ver. 2.37. As of July 09, 2024 2100 (GMT), the error code remains. ST Beta does NOT work.

I downloaded ST Stable . ST Stable WILL NOT log in to my account. The QR code request does nothing except SPAM. It is a compromised link.

Please FIX! Thank you.

**Comment by dontwantonebutneedone** [comment]
I can't login with the current just released update either. Youtube is messing around. It gets all the way through the process and says its connected to the acct, and smarttube gives me a window to select an acct and the only option is "none".

**Comment by ACR-Jeff** [comment]
Duplicate https://github.com/yuliskov/SmartTube/issues/3606

**Comment by NoAcctWanted** [comment]
No. This is NOT a "Duplicate #3606". IF the 403 error code is "fixed", the message confirms that is has NOT been fixed. I re-installed both Beta & Stable versions. It does NOT work in Beta. The Stable version contains a QR code that links to SPAM. This error is NOT fixed. It is worse.

Please actually FIX IT.

Thank you.

**Comment by drewbitt** [comment]
It is a duplicate. No one in that issue is stating it is fixed—the issue is still open. Demanding a fix on a free project in all caps to random people who don't contribute to the app won't help anything.

**Comment by NoAcctWanted** [comment]
Then you've also opened a ticket on the "stable" version. Using that does not work either. The QR Code contains bugs. That's what I am "demanding" and warning people about. Keeping that code active absolutely won't help anything.

**Comment by drewbitt** [comment]
22.35 Beta and Stable are based on identical commits. If there's any difference, it's in the build. If there's a QR code issue then an issue just for that should be raised, not mixing several things including a duplicate issue.
